main.groovy

```diff
@@ -104,6 +104,12 @@ stages {
                     ./scripts/write-parameter-file.sh "ProductVersion" ${product_version} "${WORKSPACE}/parameters/parameters.json"
                     echo "Writting product deployment region to parameter file"
                     ./scripts/write-parameter-file.sh "Region" ${product_deployment_region} "${WORKSPACE}/parameters/parameters.json"
+                    echo "Writing product release version to parameter file"
+                    ./scripts/write-parameter-file.sh "ReleaseVersion" ${release_version} "${WORKSPACE}/parameters/parameters.json"
+                    echo "Writing product distribution name to parameter file"
+                    ./scripts/write-parameter-file.sh "ProductDistributionName" ${product_distribution_name} "${WORKSPACE}/parameters/parameters.json"
+                    echo "Writing product release pack URL to parameter file"
+                    ./scripts/write-parameter-file.sh "URL" ${url} "${WORKSPACE}/parameters/parameters.json"
```

Comment:
Shall we refactor the name of this variable to $release_url

Reply:
Shall we do `custom url`? This is the custom URL we discussed right? It could be anything.

Reply:
This URL stands for the release pack URL to be clear.

Reply:
@VimukthiPerera, As I understood @VivekVinushanth is trying to do the changes in the user data itself rather than wget a custom script.

Reply:
@VimukthiPerera Shall we use the custom script approach so it will be useful for other products as well? WDYT?

Reply:
+1